src/components/custom-select.tsx

```diff
@@ -60,8 +62,9 @@ export class CustomSelect extends React.PureComponent<IProps, IState> {
 
   private renderList = () => {
     const { items } = this.props;
+    const useHeader = this.props.isHeader ? "topHeader" : "";
     return (
-      <div className={`list ${(this.state.showList ?"show" : "")}`}>
+      <div className={`list ${useHeader} ${(this.state.showList ?"show" : "")}`}>
```

Comment:
Remove `useHeader` and make the default custom-select styling match what is needed in the header.